tests/endpoints/test_operations_logs.py

```diff
@@ -0,0 +1,103 @@
+import json
+import uuid
+
+import jsonpatch
+
+from imbi.endpoints import operations_logs
+from tests import base
+
+
+class AsyncHTTPTestCase(base.TestCaseWithReset):
+
+    ADMIN_ACCESS = True
+    TRUNCATE_TABLES = [
+        'v1.operations_log',
+        'v1.environments',
+    ]
+
+    def setUp(self):
+        super().setUp()
+        self.environment = self.create_environment()
+
+    def test_operations_log_lifecycle(self):
```

Comment:
That would result in more fixtures and teardowns, since here we leverage the created records to patch, delete, etc. I also was following the pattern of other endpoint tests.

Reply:
Okay, that makes sense to me